com.unity.render-pipelines.universal/Runtime/Passes/CopyColorPass.cs

```diff
@@ -75,6 +75,13 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
             }
 
             CommandBuffer cmd = CommandBufferPool.Get();
+
+            //It is possible that the given color target is now the frontbuffer
+            if (source == renderingData.cameraData.renderer.GetCameraColorFrontBuffer(cmd))
+            {
+                source = renderingData.cameraData.renderer.cameraColorTarget;
```

Comment:
This highlights to me that we should make clear to users to not cache camera target and only access it before execute.
 To make it more clear to user we should update this pass to avoid caching the `source`. Not issue for this PR though, we can do that in follow up PR.